src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2390 @@
+      }
+    ],
+    "device": {
+      "name": "thermometer1",
+      "metadata": {
+        "scales": [
+          "Kelvin",
+          "Celsius"
+        ],
+        "max-temperature": {
+          "Kelvin": 310,
+          "Celsius": 40
+        },
+        "min-temperature": {
+          "Kelvin": 260,
+          "Celsius": -10
+        },
+        "appliable": [
+          "spring",
+          "autumn"
+        ]
+      },
+      "addresses": [
+        {
+          "type": "MAC",
+          "address": "81:ef:1a:44:7a:ff"
+        }
+      ],
+      "createdAt": "2024-10-21T18:47:45Z",
+      "updatedAt": "2024-10-21T18:47:45Z"
+    },
+    "createdAt": "2024-10-23T21:33:04.585419100Z",
+    "updatedAt": "2024-10-23T21:33:04.585419100Z"
+  }
+  ~~~
+
+- **Lookup system:** /serviceregistry/system-discovery/lookup
+  
+  _Query parameter: verbose=true_
+
+  Request body:
+  ~~~
+  {
+    "systemNames": [
+    ],
+    "addresses": [
+    ],
+    "addressType": "",
+    "metadataRequirementList": [
+      {
+        "customizable": true,
+        "type": "temperature",
+        "scales": { "op": "CONTAINS", "value": "Celsius" }
+      }
+    ],
+    "versions": [
+    ],
+    "deviceNames": [
+      "thermometer1", "thermometer2", "thermometer3", "thermometer5"
+    ]
+  }
+  ~~~
+
+  Resonse body:
+  ~~~
+  {
+    "entries": [
+      {
+        "name": "main-temperature-provider",
+        "metadata": {
+          "type": "temperature",
+          "customizable": true,
+          "scales": [
+            "Fahrenheit",
+            "Kelvin",
+            "Celsius"
+          ]
+        },
+        "version": "2.0.0",
+        "addresses": [
+          {
+            "type": "IPV4",
+            "address": "128.0.1.2"
+          },
+          {
+            "type": "MAC",
+            "address": "12:34:5a:ff:c9:44"
+          }
+        ],
+        "device": {
+          "name": "thermometer5",
+          "metadata": {
+            "scales": [
+              "Fahrenheit",
+              "Kelvin",
+              "Celsius"
+            ],
+            "max-temperature": {
+              "Kelvin": 310,
+              "Celsius": 40,
+              "Fahrenheit": 140
+            },
+            "min-temperature": {
+              "Fahrenheit": 20,
+              "Kelvin": 260,
+              "Celsius": -10
+            },
+            "appliable": [
+              "spring",
+              "summer",
+              "autumn",
+              "winter"
+            ]
+          },
+          "addresses": [
+            {
+              "type": "MAC",
+              "address": "81:ef:1a:44:7b:02"
+            }
+          ],
+          "createdAt": "2024-10-22T10:34:10Z",
+          "updatedAt": "2024-10-22T10:34:10Z"
+        },
+        "createdAt": "2024-10-23T21:43:35Z",
+        "updatedAt": "2024-10-23T21:43:35Z"
+      }
+    ],
+    "count": 1
+  }
+  ~~~
+- **Revoke system:** DELETE 
+/serviceregistry/system-discovery/revoke
```

Comment:
Important to mention here that system name comes from header (see above). Also, an example of self declared header